src/manager.js

```diff
@@ -122,15 +123,18 @@ AdManager.prototype.initGoogleTag = function () {
 */
 AdManager.prototype.fetchAmazonBids = function (elementId, gptSizes, slotName) {
   var adUnitPath = this.getAdUnitCode(),
-    slotUnit = adUnitPath + '_' + slotName;
-
+    slotUnit = adUnitPath + '_' + slotName,
+    timeoutAmount = 1e3;
```

Comment:
Let's just make this `1000` instead of `1e3`

Reply:
Good call. I've updated with the most recent commit.